test/e2e/snaps/test-snap-management.spec.js

```diff
@@ -0,0 +1,152 @@
+const { strict: assert } = require('assert');
+const { withFixtures } = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+const { TEST_SNAPS_WEBSITE_URL } = require('./enums');
+
+describe('Test Snap onOff', function () {
+  it('tests install disable enable and removal of a snap', async function () {
+    const ganacheOptions = {
+      accounts: [
+        {
+          secretKey:
+            '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+          balance: 25000000000000000000,
+        },
+      ],
+    };
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        ganacheOptions,
+        failOnConsoleError: false,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
+        // enter pw into extension
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        // open a new tab and navigate to test snaps page and connect
+        await driver.openNewPage(TEST_SNAPS_WEBSITE_URL);
+
+        // find and scroll to the correct card and click first
+        const snapButton = await driver.findElement('#connectNotification');
+        await driver.scrollToElement(snapButton);
+        await driver.delay(500);
+        await driver.clickElement('#connectNotification');
+
+        // switch to metamask extension and click connect
+        await driver.waitUntilXWindowHandles(3, 1000, 10000);
```

Comment:
`waitUntilXWindowHandles` returns the window handles. So instead of having two lines for getting the window handles we can do: `let windowHandles = await driver.waitUntilXWindowHandles(3, 1000, 10000)`

Reply:
This was fixed in 6c55ae6